Failed to build on OSX 10.9

The build of master branch (58f522ec25d845ee7734dab0ca385a8ff5ccfd3c) fails with the following error:

```
  CXXLD    filesystem_path
  CXX      handle.o
  CXXLD    handle
  CXX      keyframe.o
  CXXLD    keyframe
  CXX      node.o
/Users/admin/sources/synfig.default.master/synfig-core/test/node.cpp:130:2: error: use of overloaded operator '<<' is ambiguous (with operand types 'basic_ostream<char, std::__1::char_traits<char> >' and 'nullptr_t')
        ASSERT_EQUAL(nullptr, find_node(GUID::zero()));
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/admin/sources/synfig.default.master/synfig-core/test/test_base.h:89:3: note: expanded from macro 'ASSERT_EQUAL'
                ERROR_MESSAGE_TWO_VALUES(expected, value) \
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/admin/sources/synfig.default.master/synfig-core/test/test_base.h:68:26: note: expanded from macro 'ERROR_MESSAGE_TWO_VALUES'
        oss << "\t - expected " << a << ", but got " << b << std::endl; \
        ~~~~~~~~~~~~~~~~~~~~~~~ ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:179:20: note: candidate function
    basic_ostream& operator<<(basic_ostream& (*__pf)(basic_ostream&));
                   ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:180:20: note: candidate function
    basic_ostream& operator<<(basic_ios<char_type, traits_type>&
                   ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:182:20: note: candidate function
    basic_ostream& operator<<(ios_base& (*__pf)(ios_base&));
                   ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:195:20: note: candidate function
    basic_ostream& operator<<(const void* __p);
                   ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:196:20: note: candidate function
    basic_ostream& operator<<(basic_streambuf<char_type, traits_type>* __sb);
                   ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:880:1: note: candidate function [with _Traits = std::__1::char_traits<char>]
operator<<(basic_ostream<char, _Traits>& __os, const char* __str)
^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:834:1: note: candidate function [with _CharT = char, _Traits = std::__1::char_traits<char>]
operator<<(basic_ostream<_CharT, _Traits>& __os, const char* __strn)
^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:887:1: note: candidate function [with _Traits = std::__1::char_traits<char>]
operator<<(basic_ostream<char, _Traits>& __os, const signed char* __str)
^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:895:1: note: candidate function [with _Traits = std::__1::char_traits<char>]
operator<<(basic_ostream<char, _Traits>& __os, const unsigned char* __str)
^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:183:20: note: candidate function
    basic_ostream& operator<<(bool __n);
                   ^
/Users/admin/sources/synfig.default.master/synfig-core/test/node.cpp:134:2: error: use of overloaded operator '<<' is ambiguous (with operand types 'basic_ostream<char, std::__1::char_traits<char> >' and 'nullptr_t')
        ASSERT_EQUAL(nullptr, find_node(not_used_guid));
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/admin/sources/synfig.default.master/synfig-core/test/test_base.h:89:3: note: expanded from macro 'ASSERT_EQUAL'
                ERROR_MESSAGE_TWO_VALUES(expected, value) \
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/admin/sources/synfig.default.master/synfig-core/test/test_base.h:68:26: note: expanded from macro 'ERROR_MESSAGE_TWO_VALUES'
        oss << "\t - expected " << a << ", but got " << b << std::endl; \
        ~~~~~~~~~~~~~~~~~~~~~~~ ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:179:20: note: candidate function
    basic_ostream& operator<<(basic_ostream& (*__pf)(basic_ostream&));
                   ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:180:20: note: candidate function
    basic_ostream& operator<<(basic_ios<char_type, traits_type>&
                   ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:182:20: note: candidate function
    basic_ostream& operator<<(ios_base& (*__pf)(ios_base&));
                   ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:195:20: note: candidate function
    basic_ostream& operator<<(const void* __p);
                   ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:196:20: note: candidate function
    basic_ostream& operator<<(basic_streambuf<char_type, traits_type>* __sb);
                   ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:880:1: note: candidate function [with _Traits = std::__1::char_traits<char>]
operator<<(basic_ostream<char, _Traits>& __os, const char* __str)
^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:834:1: note: candidate function [with _CharT = char, _Traits = std::__1::char_traits<char>]
operator<<(basic_ostream<_CharT, _Traits>& __os, const char* __strn)
^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:887:1: note: candidate function [with _Traits = std::__1::char_traits<char>]
operator<<(basic_ostream<char, _Traits>& __os, const signed char* __str)
^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:895:1: note: candidate function [with _Traits = std::__1::char_traits<char>]
operator<<(basic_ostream<char, _Traits>& __os, const unsigned char* __str)
^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/ostream:183:20: note: candidate function
    basic_ostream& operator<<(bool __n);
                   ^
2 errors generated.
make[2]: *** [node.o] Error 1
make[1]: *** [check-am] Error 2
make: *** [check-recursive] Error 1
Konstantin2s-iMac:synfig.default.master admin$
```

--- a/synfig-core/test/test_base.h
+++ b/synfig-core/test/test_base.h
@@ -53,7 +53,7 @@ std::ostream& operator<<(std::ostream& os, const synfig::Vector& v)
 }
 
 // remove this operator after switch to c++17 (it is already implemented in c++17)
-#if !defined(__APPLE__) && !defined(__clang__)
+#if !defined(__APPLE__) && !defined(__clang__) || __clang_major__ == 6
 // macOS Clang toolchain has already defined this operator for all c++ versions
 std::ostream& operator<<(std::ostream& os, std::nullptr_t)
 {

